README.md

```diff
@@ -365,11 +365,12 @@ $ sudo rpm -ivh elasticsearch-1.2.1.noarch.rpm
 ```python
 SEARCH_ENGINE = 'elastic'
 ```
-- Start the Elasticsearch server and migrate the models.
+- Start the Elasticsearch server and migrate the models. Migrating search must include the production enviorment, to run 
+- locally use 'test'.
 
 ```bash
 $ invoke elasticsearch
-$ invoke migrate_search
+$ invoke migrate_search 'test'
```

Comment:
Hmm. So specifically this needs to have institutions set up, right? Or is there something else? I know we talked about this, but maybe we can catch the condition in migrate_search and handle it automatically. Alternately, perhaps a better flag, such as 'dev' or 'local'?